applications/openshift/etcd/etcd_auto_tls/oval/shared.xml

```diff
@@ -23,7 +23,7 @@
   </ind:textfilecontent54_object>
 
   <ind:textfilecontent54_state id="state_etcd_auto_tls" version="1">
-    <ind:subexpression datatype="string" operation="pattern match">^true$</ind:subexpression>
+    <ind:subexpression datatype="string" operation="pattern match">^false$</ind:subexpression>
```

Comment:
Note that the test is negated in line 9.
Check comment below in check for `ETCD_PEER_AUTO_TLS`.

Reply:
OK, I'm removing this, sorry for the error there.